lib/src/main/kotlin/net/dean/jraw/references/FlairReferences.kt

```diff
@@ -52,7 +51,16 @@ sealed class FlairReference(
      * @see SubredditReference.userFlairOptions
      */
     @EndpointImplementation(Endpoint.POST_SELECTFLAIR)
-    abstract fun updateTo(templateId: String, text: String?)
+    abstract fun updateToTemplate(templateId: String, text: String?)
+
+    /**
+     * Similar to [updateToTemplate] but sets the CSS class directly instead of using a template.
+     * The authenticated user must be a moderator of the [subreddit].
+     */
+    @EndpointImplementation(Endpoint.POST_FLAIR)
+    abstract fun updateToCssClass(cssClass: String, text: String?)
```

Comment:
First of all, updated the methods to use non-nullable input strings. Since nulls were translated into empty strings anyways, the "null" inputs are just replaced with empty string inputs.

The reason overloaded methods were used instead of default parameters is that the `meta:update` task would fail if we use default parameters with `EndpointImplementation` methods. As per documentation: 

```Note that [EndpointImplementation] methods **may not have default parameters** because of the way the Reflections library works.```
(from https://github.com/mattbdean/JRAW/blob/master/meta/src/main/kotlin/net/dean/jraw/meta/EndpointAnalyzer.kt#L15 )

I do agree that using default parameters is preferred, even though it would require java users to use only fully qualified methods. So I decided to go ahead and fix the problem with `Reflections` library and created `KotlinReflections` class which works with the methods containing default parameters. More in https://github.com/mattbdean/JRAW/pull/207/commits/0c204713e10285cff844b7101390d18ebf241fc7

Reply:
>I do agree that using default parameters is preferred, even though it would require java users to use only fully qualified methods.

Slap [`@JvmOverloads`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.jvm/-jvm-overloads/) on there and we're fine.

The KotlinReflections class is a good fix, didn't even think about that.

Reply:
Cannot add `@JvmOverloads` to abstract functions. I think we should remove default empty text parameter for flair update functions since the user should be aware that not providing this argument *does change* the flair text. In fact, it erases it. Did that in https://github.com/mattbdean/JRAW/pull/207/commits/ac5ad3782803348420875223ed27e3092122512e

Reply:
>Cannot add @JvmOverloads to abstract functions.

Huh, didn't know that.